adapter/prometheus2/BUILD

```diff
@@ -0,0 +1,37 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "prometheus.go",
```

Comment:
Also, add this adapter to the inventory2.go (there is a bazel file to link adapters in the generated inventory2.gen.go file)